CS3219-AY2425S1/cs3219-ay2425s1-project-g11
# Feat/link fe to question service

This PR aims to link frontend with the question service

- Frontend should be able to query the question service locally
- Docker compose should start frontend and backend in the same network

Dev things
- Add Axios to simplify fetch requests for frontend
- Add prettier for frontend
- Update husky for the monorepo architecture

How to test? 
- pnpm install in the necessary folders
- run `docker compose up --build`
- Open `localhost:3000`
- Verify that the questions listed are from the database
